fog-threat-model-2.1.0.md

```diff
@@ -128,8 +128,6 @@ Eve:
   (This is the current threat model for MoblieCoin consensus server.)
 
 * Otherwise, Eve obeys the SAME set of invariants as described for Oscar, with the following modifications:
-  * Eve can determine the number of new payment records retrieved during balance checking, but not precisely which payment records belong to the user.
-  * Eve can observe the ring elements used to construct a user's outgoing transaction at the time of construction.
```

Comment:
This one should not be removed -- we do not have an oblivious merkle proof service yet, but it is on the roadmap. This PR is only doing oblivious key image checks.

Reply:
ok will put back below. please confirm that is the change you wanted
 
* Eve can observe the ring elements used to construct a user's outgoing transaction at the time of construction.